File: 1000-1037 Java/SimpleCalculation.java
/*Atividade 1010
  Problem 1010*/

//Troque a classe por Main para aplicar no Beecrowd
//Change the class name to Main to submit to Beecrowd

import java.io.IOException;
import java.util.Scanner;

public class SimpleCalculation {
    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        int Cod1 = sc.nextInt();
        int Qnt1 = sc.nextInt();
        double Val1 = sc.nextDouble();
        int Cod2 = sc.nextInt();
        int Qnt2 = sc.nextInt();
        double Val2 = sc.nextDouble();
        double Total = (double) Val1 * Qnt1 + Val2 * Qnt2;
        System.out.printf("VALOR A PAGAR: R$ %.2f\n", Total);
        sc.close();
    }
 
}

File: 1000-1037 Python/SimpleCalculation.py
#Atividade 1010
#Problem 1010

Cod1, Qnt1, Val1 = input().split()
Cod2, Qnt2, Val2 = input().split()

Qnt1 = int(Qnt1)
Val1 = float(Val1)
Qnt2 = int(Qnt2)
Val2 = float(Val2)

Total = Qnt1 * Val1 + Qnt2 * Val2
print(f"VALOR A PAGAR: R$ {Total:.2f}")